Comp2023/src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -184,40 +182,32 @@ public void simulationPeriodic( )
     // This method will be called once per scheduler run during simulation
 
     // Set input flywheel voltage from the motor setting
-    m_elbowMotorSim.setBusVoltage(RobotController.getInputVoltage( ));
-    m_elbowSim.setInput(-m_elbowMotorSim.getMotorOutputLeadVoltage( ));
 
     m_wrist16MotorSim.setBusVoltage(RobotController.getInputVoltage( ));
```

Comment:
We might want to leave numbers out of names, but if you do want to keep it, I think the ID changed to 17.